src/ovms.h

```diff
@@ -27,7 +27,10 @@ typedef struct OVMS_Status_ OVMS_Status;
 typedef struct OVMS_ServerSettings_ OVMS_ServerSettings;
 typedef struct OVMS_ModelsSettings_ OVMS_ModelsSettings;
 
-// TODO reuse this in precision.hpp
+// OVMS_DataType
+//
+// Tensor and parameter data types recognized by OVMS
```

Comment:
Do we recognize other parameter types than int and string?

Reply:
Right now we don't use parameters at all. THey are placeholder for Stateful support & reusing C-API inference by both TFS & KFS

Reply:
Then the user can feel misguided that we will end up supporting all the precisions for parameters

Reply:
It's rather question of which parameters we actually support. For now candidates are sequence_id, sequence_control_input, & one parameter for KFS informing where to put the  buffer content. So right now we limit the support of parameters by not supporting any in preview (we are just not using those), and we can whitelist parameter and support them one by one as we have to.